### PR TITLE
<Timeline/> - allow passing a node instead of string only in the item…

### PR DESCRIPTION
**Pull request by sivanharel**
….label prop

### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5883/

**Comment by sivanharel** [comment]
@milkyfruit - I made the changes in order to allow passing node in the label prop.
Do we want to add his example to the storybook?
`https://www.wix.com/pages/wix-style-react/?path=/story/introduction-playground--playground`

File: src/Timeline/Timeline.uni.driver.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ export const timelineDriverFactory = (base, body) => {
   return {
     ...baseUniDriverFactory(base, body),
 
+    /** Checks that a node with the provided predicate that is used for the Label exists */
+    hasLabelNode: async predicate => await base.$(predicate).exists(),
```
Comment (EgoziE) [inline src/Timeline/Timeline.uni.driver.js:9]: I don't think this is a good testkit API
If someone needs to target a node, they can use their own data-hook.

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@sivanharel we can add this example:

- [x] **Custom Content**
Component's label allows to insert any content to build custom layouts.
```
() => {
  const items = [
    {
      label: (
        <Layout cols={1} gap={0}>
          <Heading appearance="H6">Review Request Date</Heading>
          <Text size="small">Jul 05, 2020 at 11:00 AM</Text>
          <Box marginTop={2}>
            <Badge type="outlined">Waiting for review</Badge>
          </Box>
          <Box marginTop={3}>
            <Text size="small">
              Ask your client for a review directly from here. Send this
            </Text>
          </Box>
          <Box>
            <TextButton size="small">Request Review link</TextButton>
          </Box>
        </Layout>
      ),
    },
    {
      label: (
        <Layout cols={1} gap={6}>
          <Heading appearance="H6">Submitted Date</Heading>
          <Text size="small">Jul 04, 2020 at 09:00 AM</Text>
          <Box marginTop={3}>
            <Heading appearance="H6">Website Url</Heading>
          </Box>
          <Box>
            <TextButton size="small">www.mycurrentlywebsite.com</TextButton>
          </Box>
        </Layout>
      ),
    },
  ];

  return (
    <Card>
      <Card.Content>
        <Timeline items={items} />
      </Card.Content>
    </Card>
  );
};
```

File: src/Timeline/Timeline.uni.driver.js (hunk 1 above)
Comment (shlomitc) [reply src/Timeline/Timeline.uni.driver.js:9]: it makes sense if it's more specific
`await base.$('[data-hook="some-internal-part"]').$(predicate).exists()`

This is more accurate and does give value to make sure it is in the right place

**Review by EgoziE — APPROVED** [review]

**Pull request merged**